gMax:
- 1.0
- 1.0
- 2.9321531433504733
- 3.0
- 8.0
gMin:
- -1.0
- -1.0
- -3.141592653589793
- 0.0
- -8.0
gN:
- 50
- 50
- 30
- 20
- 20
pMax:
- 1.5
pMin:
- -1.5
uMax:
- 0.5
- 6
uMin:
- -0.5
- -6
v: 0.1
x0:
- 3
- 3
- 0
- 0
- 0
